curp/src/server/curp_node.rs

```diff
@@ -51,6 +51,9 @@ pub(super) enum CurpError {
     /// Storage error
     #[error("storage error, {0}")]
     Storage(#[from] StorageError),
+    /// Get applied index error
+    #[error("get applied index error, {0}")]
+    GetAppliedIndex(String),
```

Comment:
How about `Internal(String)` and move the `get applied index error` below? It's too specific.

Reply:
Sure